Containerfile

```diff
@@ -40,6 +40,11 @@ COPY --from=debian-build pulp-fixtures/fixtures /usr/share/nginx/html
 
 # turn on autoindex
 RUN sed -i -e '/location \/ {/a autoindex on\;' /etc/nginx/conf.d/default.conf
+# tune nginx config file for auth token
+RUN mkdir -p /pulp-fixtures/common
+ADD common /pulp-fixtures/common
+RUN sed -i -e '/http {/r /pulp-fixtures/common/nginx_auth_token_map' /etc/nginx/nginx.conf
+RUN sed -i -e '/server {/r /pulp-fixtures/common/nginx_auth_token_cond' /etc/nginx/conf.d/default.conf
```

Comment:
Maybe, at this point we should just copy our version of nginx.conf into the container image. This sed surgery start to look adventurous to me.
Do not get me wrong. I love a good sed script. 👨‍💻 But it is also a bit dangerous.